libraries/GROOT/GRootObjectManager.cxx

```diff
@@ -267,12 +267,12 @@ TH1 *GRootObjectManager::GetLast1D(TObject *object) {
 
 
 
-TH2    *GRootObjectManager::GetNext2D(TObject *object) { }
-TH2    *GRootObjectManager::GetLast2D(TObject *object) { }
+TH2    *GRootObjectManager::GetNext2D(TObject *object) { return 0; }
+TH2    *GRootObjectManager::GetLast2D(TObject *object) { return 0; }
```

Comment:
Yeah I'm just trying to make it so apple doesn't barf everywhere even during a successful compilation. It looks like that might take ROOT6 though because of the dictionary generation.